streamer/src/nonblocking/simple_qos.rs

```diff
@@ -88,8 +92,18 @@ impl SimpleQos {
     > {
         let remote_addr = connection.remote_address();
 
+        // this will never overflow u32 for reasonable MAX_RTT
+        let rtt = connection.rtt().clamp(MIN_RTT, MAX_RTT).as_millis() as u32;
+        let max_streams_in_flight = (self.config.max_streams_per_second as u32).saturating_mul(rtt)
+            / 1000
+            * STREAMS_IN_FLIGHT_MARGIN;
+        // for very low values of max_streams_per_second, prevent connections from having zero
+        // streams in flight
+        let max_streams_in_flight = max_streams_in_flight.max(STREAMS_IN_FLIGHT_MARGIN);
+        connection.set_max_concurrent_uni_streams(VarInt::from_u32(max_streams_in_flight));
```

Comment:
It looks like the receive window remained to be set to `PACKET_DATA_SIZE` (default option in `configure_server()`, thus limiting votes TPS to 1 per RTT.